packages/ui/src/coreModules/resourceManager/components/ResourceManager/MainColumn/item/ActionBars/Base/index.js

```diff
@@ -53,6 +54,7 @@ const propTypes = {
 }
 const defaultProps = {
   loadingDelete: undefined,
+  nestedItem: undefined,
```

Comment:
you have both defaultProp and it isRequired. maybe it should not be required and default to {} since that's what it does in render

Reply:
Its fixed in a later commit